### PR TITLE
Feature/updated logs file size

### PR DESCRIPTION
**Pull request by deoyani**
This branch addresses issues of logging more information about each request to distribution service.

1. BundlePlan request
This request logs summary of the requested files when service creates a plan to download
And
Details of each file requested including filepath, file size and recordid 
2. BundleRequest
This part logs actual download requests per file 
filepath, recordid, filesize, bundlename etc
and bundles request
bundlename, size, status etc

These log messages are useful for the Metrics project to read the logs for our understanding of data usage and also to display information on the individual landing pages.

### CODE REVIEW

File: src/main/java/gov/nist/oar/distrib/datapackage/DownloadBundlePlanner.java [hunk 1]
```diff
@@ -140,10 +157,64 @@ public BundleDownloadPlan getBundleDownloadPlan() throws DistributionException,
...
+	 * ** No specing after commas. Keep semicolons
+	 */
+	private void printLogsSummary() {
+	    try {
+
+	    String printLog = "";
+	    int fileList = 0 ;
+	    if(inputfileList != null ) fileList = inputfileList.length;
+		
+	    logger.info("BundlePlan Summary: RequestId:"+requestId+",Status :"+this.status+",Total Size:"+totalRequestedFileSize+",Bundles:"+ bundleCount
```
Comment (RayPlante) [inline src/main/java/gov/nist/oar/distrib/datapackage/DownloadBundlePlanner.java:178]: Do you mean to have  a space after ",Status"?

**Review by RayPlante — CHANGES_REQUESTED** [review]
I tested under oar-docker, and downloads continue to work, and I confirmed that bundle plan information is being written to the log.  

(There was a problem showing up in the log with handling user-cancelled downloads, but I feel this is out-of-scope for this PR; I'll create a ticket.)

I have a few minor style and documentation issues to mention; please comment on or update the code as appropriate on each of the following:
  * the README.md file has some commentary added about logging (thanks!); however, is the information still applicable?  I do not see any evidence in the code that the three files mention are created.  Plus, please look at the formatting; the line-breaks are not doing what you think they should be doing.  
  * `FileRequestLog.java`:  this is a new file; please fix these formatting issues:
    * change all indentation to 4 spaces.
    * fill out or remove all empty (auto-generated, I'm guessing) documentation blocks, including those with blank @param tags.
  * `DownloadBundlePlanner.java`:  fill out or remove empty documentation block on constructor
  * `BundleDownloadPlan.java`:  remove empty documentation blocks on private member variables
  * I notice that many log messages feature spacing that is not optimal for reading.  Many of these, I assume, are optimized for parsing, which is fine.  I've made some code annotations where I suspect the spacing is unintended.  Please fix or confirm that the spacing is as you intended.

**Comment by RayPlante** [comment]
Also, please fix failing unit tests.  (See end of [travis results](https://travis-ci.org/github/usnistgov/oar-dist-service/builds/756423690?utm_source=github_status&utm_medium=notification), line 4613.)

**Comment by deoyani** [comment]
Ray,  I had updated failing tests, they are working fine on my machine no errors. I think travis is failing to build.

**Comment by RayPlante** [comment]
One on-push travis build failed but the update-to-pr build passed; this suggests that the test itself is unstable; perhaps one of the requested files is not reliably available?  I see that you fixes just manipulates the number of the requested files are expected in the returned bundle.  What is the intention of `getBundleDownloadPlan2Test`?  Which file is not getting found?  Do you need to pick a more reliable target file?

**Review by RayPlante — APPROVED** [review]

**Pull request merged**